### ISSUE TITLE
Female and Male 

### ISSUE THREAD

**Issue by GangstaVik** [MrSimbax/cp2077-mod-wardrobe-items-adder#13]
The script gives both Vs stuff so it gets confusing when using lots of clothing mods ... so please if there is tell me how to filter them based on V's genre or add this function to the script... ty 
(sorry, Ik this is not really an issue but more of a function req ... I've seen on nexus that you might miss comments there )

**Comment by MrSimbax** [comment]
Hmm, it's been awhile since I last played, and I wasn't really investigating how gender works in this game while working on the mod, but I don't recall there being a way to distinguish female and male clothes. From what I played and seen from random screenshots, IIRC male V can still find/buy/wear female clothing, f.e. dresses and skirts. It's just that some clothes, like T-shirts, have probably two different 3D models depending on V's body type, but they're still treated as the same item.

One solution would be to filter by their ID, e.g. if all skirts are prefixed by `FormalSkirt_`, then we could treat any item that has this prefix as for female, and thus skip these items if the user wishes so. But mods don't have to follow that naming convention (I'm not even sure if all vanilla items do, would have to check), so I'd rather avoid that.

It might be worth noting that there are *some* weird duplicates in vanilla like `Items.Q301_Corpo_WA_Face` and `Items.Q301_Corpo_MA_Face` (I guess `MA` variant is for male V, and `WA` is for female V? not sure), but both items are referencing the same appearance, so I don't know why it was done like this, and why only some (quest?) items have these variants. Majority of items don't include anything like this in their ID though, so this definitely won't be that useful as a filter even for vanilla.

Of course, hardcoding the list of female-only clothes is an option, but I am certainly not going to do that as the whole point of this mod is to add everything automatically. Even if, the list approach would only work for vanilla items, unless the user would be willing to update the lists manually for the specific mods they use. But if they're willing to go that far, they might as well just copy-paste their list to add only items they want, instead of using the "add all clothes" button.

To sum up, sorry to disappoint, but it's unlikely I'll implement anything like this as there doesn't seem to be an easy and reliable way to differentiate male and female clothes automatically.

**Comment by GangstaVik** [comment]
> Hmm, it's been awhile since I last played, and I wasn't really investigating how gender works in this game while working on the mod, but I don't recall there being a way to distinguish female and male clothes. From what I played and seen from random screenshots, IIRC male V can still find/buy/wear female clothing, f.e. dresses and skirts. It's just that some clothes, like T-shirts, have probably two different 3D models depending on V's body type, but they're still treated as the same item.
> 
> One solution would be to filter by their ID, e.g. if all skirts are prefixed by `FormalSkirt_`, then we could treat any item that has this prefix as for female, and thus skip these items if the user wishes so. But mods don't have to follow that naming convention (I'm not even sure if all vanilla items do, would have to check), so I'd rather avoid that.
> 
> It might be worth noting that there are _some_ weird duplicates in vanilla like `Items.Q301_Corpo_WA_Face` and `Items.Q301_Corpo_MA_Face` (I guess `MA` variant is for male V, and `WA` is for female V? not sure), but both items are referencing the same appearance, so I don't know why it was done like this, and why only some (quest?) items have these variants. Majority of items don't include anything like this in their ID though, so this definitely won't be that useful as a filter even for vanilla.
> 
> Of course, hardcoding the list of female-only clothes is an option, but I am certainly not going to do that as the whole point of this mod is to add everything automatically. Even if, the list approach would only work for vanilla items, unless the user would be willing to update the lists manually for the specific mods they use. But if they're willing to go that far, they might as well just copy-paste their list to add only items they want, instead of using the "add all clothes" button.
> 
> To sum up, sorry to disappoint, but it's unlikely I'll implement anything like this as there doesn't seem to be an easy and reliable way to differentiate male and female clothes automatically.

Okay dude understood. Thank you for reading my message and for the time you took answering me. By the way I remember noticing the WA and MA in the .apps names to have them separated from each other.
Ty anyway <3 have a nice day <3
